src/api_server/swagger/firecracker.yaml

```diff
@@ -1097,7 +1116,6 @@ definitions:
   SnapshotLoadParams:
```

Comment:
Should add here that exactly one of the two `mem_*` parameters must be present in the body of the request.